### PR TITLE
improve logging timeout error message

### PR DESCRIPTION
**Pull request by 0xAX**
Now it should put radius attributes to journal meta fields instead of
writing them into message attribute.

### CODE REVIEW

File: src/eradius_server.erl [hunk 1]
```diff
@@ -325,6 +326,12 @@ apply_handler_mod(HandlerMod, HandlerArg, Request, NasProp) ->
             {reply, EncReply,{Request#radius_request.cmd, ReplyCmd}};
         noreply ->
             {discard, handler_returned_noreply};
+        {error, timeout} ->
+            S = {NasProp#nas_prop.nas_ip, NasProp#nas_prop.nas_port, Request#radius_request.reqid},
+            NAS = eradius_lib:get_attr(Request, ?NAS_Identifier),
+            NAS_IP = inet_parse:ntoa(NasProp#nas_prop.nas_ip),
+            lager:error(eradius_log:collect_meta(S, Request), "timeout after waiting for response from RADIUS NAS: ~s ~s", [NAS, NAS_IP]),
```
Comment (thz) [inline src/eradius_server.erl]: Can we also add information about what we were waiting for?
example: timeout waiting for response [...] to Access-Request(1) [...]
The "Access-Request(1)" is quite relevant for operations. (what kind of request / request id)

Comment (0xAX) [inline src/eradius_server.erl]: yes, will add

**Comment by 0xAX** [comment]
@thz updated

**Comment by 0xAX** [comment]
Example of output:

```
Fri 2016-11-18 18:20:52.779426 +06 [s=a74a87b9afa64080bb943a5b84cb5a23;i=773f;b=3c5b1161a678467598a37635936e3a83;m=36c449f74;t=541925651b60a;x=9019dc301e42539c]
    _BOOT_ID=3c5b1161a678467598a37635936e3a83
    _MACHINE_ID=f3aa46ca75344bdd841aa27e3c9d3705
    _HOSTNAME=localhost
    PRIORITY=3
    _TRANSPORT=journal
    _CAP_EFFECTIVE=0
    _GID=1000
    _SYSTEMD_OWNER_UID=1000
    _SYSTEMD_SLICE=user-1000.slice
    _UID=1000
    _SYSTEMD_CGROUP=/user.slice/user-1000.slice/session-c2.scope
    _SYSTEMD_SESSION=c2
    _SYSTEMD_UNIT=session-c2.scope
    SYSLOG_IDENTIFIER=beam.smp
    _COMM=beam.smp
    _EXE=/usr/local/lib/erlang/erts-8.0/bin/beam.smp
    _CMDLINE=/usr/local/lib/erlang/erts-8.0/bin/beam.smp -- -root /usr/local/lib/erlang -progname erl -- -home /home/alex -- -pa /home/alex/work/elixir/bin/../lib/eex/ebin /home/alex/work/eli
    ERL_NODE=nonode@nohost
    ERL_APPLICATION=eradius
    REQUEST_TYPE=Access-Request
    ERL_MODULE=eradius_server
    ERL_FUNCTION=apply_handler_mod
    ACCT_TERMINATE_CAUSE=User-Request
    TP_RELATED_SESSION_ID=262019876543210
    NAS_IDENTIFIER=SCG
    ERL_PID=<0.1157.0>
    REQUEST_ID=6
    TP_ACCESS_CLASS_ID=Online
    TP_NAT_IP_ADDRESS=10.0.0.0
    ERL_LINE=323
    _PID=25640
    FRAMED_IP_ADDRESS=10.0.183.175
    EVENT_TIMESTAMP=2016-11-18T12:20:52
    TP_NAT_PORT_START=2166
    TP_NAT_PORT_END=2176
    ACCT_MULTI_SESSION_ID=389f434d-3e17-48b9-8696-cb5539d17ca5
    ACCT_SESSION_ID=35cb1e6f-8d68-4ab1-98b5-011bff129931
    MESSAGE=timeout after waiting for response to Access-Request(6) from RADIUS NAS: SCG 127.0.0.1
    _SOURCE_REALTIME_TIMESTAMP=1479471652779426

```

**Pull request merged**